ansible_events_ui/db/utils/lostream.py

```diff
@@ -0,0 +1,227 @@
+from __future__ import annotations
+
+import logging
+from io import UnsupportedOperation
+from typing import Tuple, Union
+
+from fastapi import Depends
+from sqlalchemy.ext.asyncio import AsyncSession
+
+from ansible_events_ui.db.dependency import get_db_session
+
+LOG = logging.getLogger(__name__)
+
+
+# Default read buffer size is
+CHUNK_SIZE = 1024**2
```

Comment:
Since the constant name requires additional comment, to describe its designation, maybe better to name it `READ_BUFFER_SIZE` ?

Reply:
"Chunk" seems to be a well used term for a buffer size in a streaming context.

Reply:
The naming preferably to be self-descriptive. The comment for the variables indicates that the name is not self-descriptive enough.